app/lib/database/index.js

```diff
@@ -59,6 +50,17 @@ class DB {
 		return this.databases.serversDB;
 	}
 
+	setServersDB() {
```

Comment:
Maybe I don't need to change this.
The new instance will be created only after the first import of this file, so, I can make the copy of the database file before the `restore` on `app/index`.

Reply:
Unfortunately it's not possible :(